docs/upgrade.html

```diff
@@ -15,8 +15,23 @@
  limitations under the License.
 -->
 
+
+
 <h3><a id="upgrade" href="#upgrade">1.5 Upgrading From Previous Versions</a></h3>
 
+<h4><a id="upgrade_10_1" href="#upgrade_10_1">Upgrading from 0.10.0.X to 0.10.1.0</a></h4>
+
+0.10.1.0 is fully compatible with 0.10.0.X. The upgrade can be done one broker at a time by simply bringing it down, updating the code, and restarting it.
```

Comment:
How can we say it is fully compatible and then say that there are `Potential breaking changes`?